### ISSUE TITLE
Don't commit home folder

### ISSUE THREAD

**Issue by tetienne** [andsens/homeshick#184]
Hi, 

When you create a castle, the home folder is always present. Can we have directly the content of the home folder in the castle?

**Comment by andsens** [comment]
Nope, that would break all existing castles of all users, unless you introduced some weird config var. And you wouldn't then be able to track things like e.g. prezto or oh-my-zsh.
